microsoft/botframework-solutions
# [TypeScript][Botskills] Refactor in Mocks folder

## Purpose
*What is the context of this pull request? Why is it being done?*
The `mocks` folder is disorganized, it could be separated into the following folders:
- `success`: for the correctly resources (for example, dispatch, lu) that are used in the tests
- `skills`: to contain the different test manifests
- `virtualAssistant` : to contain the `skills` files

## Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp?*

- Change the structure of the `mocks` folder
- Rename all the files to improve the consistency
- Update connect/disconnect/update/refresh/list tests
- Fix message typos

*Include illustrative screenshots for context if applicable.*
![image](https://user-images.githubusercontent.com/11904023/60543531-e766ce00-9cec-11e9-9aca-6ffa763e2d3e.png)

## Tests
*Is this covered by existing tests or new ones? If no, why not?*
We updated the tests to cover this changes. (connect/disconnect/update/refresh/list tests)

## Testing Steps
1. Go to `botframework-solutions\tools\botskills`
1. Execute `npm install`
1. Execute `npm run build`
1. Execute `npm run test` and check that all the tests are passing successfully

![image](https://user-images.githubusercontent.com/11904023/60543953-0ade4880-9cee-11e9-91a5-dbec2dcbe09e.png)

## Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-
